### PR TITLE
Adding ability to start and stop rosbag recording on command

### PR DESCRIPTION
**Pull request by pvechersky**
Adding a service to the gazebo_bag_plugin to start and stop the recording of the bag file on command if the option is enabled

### CODE REVIEW

File: rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_bag_plugin.h [hunk 1]
```diff
@@ -73,7 +76,10 @@ class GazeboBagPlugin : public ModelPlugin {
         frame_id_(kDefaultFrameId),
         link_name_(kDefaultLinkName),
         bag_filename_(kDefaultBagFilename_),
+        recording_service_name_(kDefaultRecordingServiceName),
         rotor_velocity_slowdown_sim_(kDefaultRotorVelocitySlowdownSim),
+        wait_to_record_(kDefaultWaitToRecord),
+        is_recording_(false),
```
Comment (ffurrer) [inline rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_bag_plugin.h]: can you add this as a constant, like the other default values?

**Comment by pvechersky** [comment]
@ffurrer 

Done :)

**Comment by burrimi** [comment]
lgtm.

**Comment by pvechersky** [comment]
Sorry, one last commit for this in order to add the new parameter to all the quad base description files so that all the models still launch, and also setting the default value such that the default behavior is the old one where bag is recorded from the very beginning.

**Pull request merged**